config.yml

```diff
@@ -63,27 +63,58 @@ frame_extraction:
 
 training_pipeline:
   model_parameters:
-    img_size: 1024
-    batch: -1
+    img_size: 1280
+    batch: 0.85
+    epochs: 500
+    patience: 75
+    n_classes: 5
+    cos_lr: True
+    dropout: 0.0
+    seed: 111
+    box: 7.893
+    cls: 7.881
+    dfl: 5.767
+    name_classes: ["person", "license plate", "container", "mobile toilet", "scaffolding"]
+    rect: False
+  inputs:
+    datastore_path: "first_official_training_dataset_oor"
+    training_data_rel_path: "processed-merged-batches-first-official-training-dataset-oor"
+    model_weights_rel_path: "model"
+    model_name: "best_yolov8m_coco_290524_old_oor_nc_5.pt"
+    wandb_api_key: "3913776b1d82ae9f4f08090917114a0ad61ecfcf"
+    wandb_mode: "online"
+  outputs:
+    project_datastore_path: "first_official_training_dataset_oor"
+    project_rel_path: "model"
+
+sweep_pipeline:
+  model_parameters:
+    img_size: 1280
+    batch: 0.85
     epochs: 100
-    patience: 25
-    n_classes: 3
-    name_classes: ["person", "license plate", "container"]
+    patience: 100
+    n_classes: 5
+    name_classes: ["person", "license plate", "container", "mobile toilet", "scaffolding"]
+    rect: False
+  sweep_trials: 10
   inputs:
-    datastore_path: "first_training_dataset_oor"
-    training_data_rel_path: "first-training-dataset-oor"
+    datastore_path: "first_official_training_dataset_oor"
+    training_data_rel_path: "processed-merged-batches-first-official-training-dataset-oor"
     model_weights_rel_path: "model"
-    model_name: "yolov8m.pt"
+    model_name: "best_yolov8m_coco_290524_old_oor_nc_5.pt"
+    wandb_api_key: "3913776b1d82ae9f4f08090917114a0ad61ecfcf"
+    wandb_mode: "online"
+    sweep_config: "sweep_config.json" # needs to be in the same directory as model_weights_rel_path
   outputs:
-    project_datastore_path: "first_training_dataset_oor"
+    project_datastore_path: "first_official_training_dataset_oor"
     project_rel_path: "model"
 
 logging:
-  loglevel_own: INFO  # override loglevel for packages defined in `own_packages`
+  loglevel_own: DEBUG  # override loglevel for packages defined in `own_packages`
   own_packages: ["__main__", "objectherkenning_openbare_ruimte", "convert_annotations"]
   basic_config:
     # log config as arguments to `logging.basicConfig`
     level: WARNING
     format: "%(asctime)s|||%(levelname)-8s|%(name)s|%(message)s"
     datefmt: "%Y-%m-%d %H:%M:%S"
-  ai_instrumentation_key: "APPLICATION_INSIGHTS_CONNECTION_STRING"
+  ai_instrumentation_key: ""
```

Comment:
This needs to be ai_instrumentation_key: "{AI_INSTRUMENTATION_KEY}"
After the split we can clean it.